tests/test_flex_counters.py

```diff
@@ -0,0 +1,83 @@
+import time
```

Comment:
@shlomibitton  could you please provide description of the flows to be tested in this unit test? it will help to ensure all cases are handled as part of the ut

Reply:
Added the test flow and description to the PR.